src/components/Button/index.tsx

```diff
@@ -8,9 +8,28 @@ import type { Theme } from '@emotion/react'
 type ButtonStyleType = typeof BUTTON_STYLE_KEYS[keyof typeof BUTTON_STYLE_KEYS]
 type ButtonSize = 'small' | 'medium' | 'large'
 export interface ButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  /**
+   * Button의 스타일을 정합니다.
+   * @type "ghost" | "outline" | "outlineDisabled" | "solidDisabled" | "solidPrimary" | "solidSub" | undefined
+   */
```

Comment:
```suggestion
   * Button의 보여질 형태를 정합니다.
   * @type "ghost" | "outline" | "outlineDisabled" | "solidDisabled" | "solidPrimary" | "solidSub" | undefined
   */
```
styleType은 보여질 형태로 정의하는 건 어떠신가요?!

Reply:
[요기](https://github.com/price-offer/offer-design-system/pull/82/commits/d25c1a5824bdc6dc9f826b2f031f9ebaa2c886d7)에 반영 완료! 🥰